Test/DependencyCollector/FunctionalTests/FuncTest/IIS/IisApplicationPool.cs

```diff
@@ -96,6 +96,7 @@ public IisApplicationPool(string poolName, string managedRuntimeVersion = Defaul
             catch (Exception ex)
             {
                 Trace.TraceError("Exception occured while creating IIS Application Pool: {0}. The exception thrown was {1}", poolName, ex.Message);
+                throw ex;
```

Comment:
btw - this is simply a fix to makes tests fail instantly if initialize has issues